drivers/eeprom/eeprom_st25dv.c

```diff
@@ -43,7 +43,86 @@ LOG_MODULE_REGISTER(eeprom_st25dv);
 #define ST25DV_REG_IC_REF      0x0017
 #define ST25DV_REG_UUID        0x0018
 #define ST25DV_REG_IC_REV      0x0020
-#define ST25DV_REG_I2C_PWD     0x0900
+#define ST25DV_REG_I2C_PASSWD  0x0900
+#define ST25DV_REG_I2C_SSO_DYN 0x2004
+
+/** @brief Length of the MEM_SIZE field of the ST25DV */
+#define ST25DV_MEM_SIZE_LENGTH   (2u)
```

Comment:
There is no reason, unless you do not know what you are doing, to put a single token in parentheses.

Reply:
Old habit sorry. I'm not too familiar (yet) with Zephyr coding rules. Code updated to remove the (uint8_t) on defined and the parentheses